docs/section23/Rule23-10.md

```diff
@@ -0,0 +1,62 @@
+
+# Airside System - Rule 23-10  
+
+**Schema Version:** 0.0.10  
+**Mandatory Rule:** True  
+**Rule ID:** 23-10  
+**Rule Description:** System 5-8 and 11 - part load VAV fan power shall be modeled using either method 1 or 2 in Table G3.1.3.15. This rule will only validate data points from Method-1 Part-load Fan Power Data. However, both methods are equivalent. When modeling inputs are based on Method 2, values should be converted to Method 1 when writing to RMD.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**90.1 Section Reference:** Section G3.1.3.15 VAV Fan Part-Load Performance (Systems 5 through 8 and 11)  
+**Data Lookup:** Table G3.1.3.15  
+**Evaluation Context:** Building  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-5, 6, 7, 8, 11.1, 11.2, 7a, 8a, 11.1a, 11.2a, 5b, 6b, 7b, 8b, 11b, 7c, 11c.  
+
+**Function Calls:**  
+
+1. get_baseline_system_types()  
+2. is_baseline_system_type()
+
+**Applicability Checks:**  
+
+- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-5, 6, 7, 8, 11.1, 11.2, 7a, 8a, 11.1a, 11.2a, 5b, 6b, 7b, 8b, 11b, 7c, 11c, continue to rule logic: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-5", "SYS-6", "SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-7A", "SYS-8A", "SYS-11.1A", "SYS-11.2A", "SYS-5B", "SYS-6B", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-7C", "SYS-11C"]): CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+## Rule Logic:  
+
+- For each building segment in B_RMR: `for building_segment_b in B_RMR...building_segments:`
+
+  - Get HVAC system: `hvac_b = building_segment_b.heating_ventilation_air_conditioning_systems`
+
+    - Check if HVAC system is type 5, 6, 7, 8, 11.1, 11.2, 7a, 8a, 11.1a, 11.2a, 5b, 6b, 7b, 8b, 11b, 7c, 11c: ``if any(is_baseline_system_type(hvac_b, sys_type) == TRUE for sys_type in ["SYS-5", "SYS-6", "SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-7A", "SYS-8A", "SYS-11.1A", "SYS-11.2A", "SYS-5B", "SYS-6B", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-7C", "SYS-11C"]):`
+
+      - Get fan system serving HVAC system: `fan_system_b = hvac_b.fan_system`
+
+        - For each supply fan in fan system: `for supply_fan_b in fan_system_b.supply_fans:`
+
+          **Rule Assertion:**
+
+          - Case 1: For supply fan in fan system, if fan part-load VAV is modeled as per Table G3.1.3.15: `if supply_fan.output_validation_points == {0: 0, 
```

Comment:
Based on schema note, "Airflow is input to each validation point and energy output is the result." So, in order to verify the match between Table G3.1.3.15 and fan validation point in the RMD, we need to divide the first value in fan.validation_point pair by fan.design_airflow, and the second value in each pair by fan.design_electric_power.

Reply:
I have also submitted schema issue for this.

Reply:
Link to Schema issue https://github.com/open229/ruleset-model-description-schema/issues/109